.github/workflows/docker_image.yaml

```diff
@@ -0,0 +1,42 @@
+name: build docker image
+
+on:
+  push:
```

Comment:
Why not `pull_request`? Now, we hope that every change should be a PR

Reply:
It seems that tag can't be a PR? Github would auto push a new version when publishing the new release.